docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -293,30 +293,40 @@ Update of the software will be managed by the swupd tool::
 Latest Releases via Pip
 +++++++++++++++++++++++
 
-Ansible can be installed via "pip", the Python package manager.  If 'pip' isn't already available in
-your version of Python, you can get pip by::
+Ansible can be installed via ``pip``, the Python package manager.  If ``pip`` isn't already available in
+your version of Python, you can get ``pip`` by::
 
-   $ sudo easy_install pip
+    $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    $ sudo python get-pip.py
 
 Then install Ansible with [1]_::
 
-   $ sudo pip install ansible
+    $ sudo pip install ansible
 
 Or if you are looking for the latest development version::
 
-   $ pip install git+https://github.com/ansible/ansible.git@devel
+    $ sudo pip install git+https://github.com/ansible/ansible.git@devel
 
-If you are installing on macOS Mavericks, you may encounter some noise from your compiler.  A workaround is to do the following::
+If you are installing on macOS Mavericks, you may encounter some noise from your compiler. A workaround is to do the following::
 
-   $ sudo CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install ansible
+    $ sudo CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install ansible
 
-Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally. Do not use easy_install to install Ansible directly.
+If you need to use the ``paramiko`` connection plugin, install the required module[2]_::
+
+    $ sudo pip install paramiko
+
+Ansible can also be installed in the user directory by using ``--user``::
+
+    $ python get-pip.py --user
```

Comment:
since you're in the neighborhood- should we make this the recommended method instead of global (since pip does the same)?

Reply:
I did remove the line about "just install in globally" that seemed a bit glib. I'll just reverse this suggestion, ``--user`` by default but ``sudo`` if you want it globally with big warning.